### PR TITLE
ACMS-768: Rewritten the install finished method.

### PR DESCRIPTION
**Pull request by panshulK**
**Motivation**
* The UI install is throwing 403 Forbidden error.
Fixes #841 

**Proposed changes**
* Updated the code that runs upon install_finish. 

**Alternatives considered**
* Was able to get the issue fixed by changing 2 things - 
** Changed Drupal version from 9.2.0 -> 9.1.9
** Updated the code in the method called by install_finish installation task.

* Screenshots attached below
Screenshot 2021-07-02 at 9 24 24 PM
Screenshot 2021-07-02 at 9 33 17 PM

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: acquia_cms.profile [hunk 1]
```diff
@@ -38,6 +38,7 @@ function acquia_cms_install_tasks_alter(array &$tasks) {
   // This code helps capture time right when the drupal bootstrap happens.
   // The pre start function calls another method that performs the actual logic.
   $tasks['install_bootstrap_full']['function'] = 'acquia_cms_pre_start';
+  $tasks['install_finished']['function'] = 'acquia_cms_install_ui_kit_finished';
```
Comment (vishalkhode1) [inline acquia_cms.profile:41]: @panshulK This is causing issue. We don't need this. Right ?
cc: @chandan-singh7929

Comment (panshulK) [reply acquia_cms.profile:41]: @vishalkhode1  The issue with this function has been resolved in this PR itself. We need the function back. This was accidently removed in @chandan-singh7929 's PR for 9.2.0 upgrade fix.

File: acquia_cms.profile [hunk 2]
```diff
@@ -266,10 +297,19 @@ function acquia_cms_install_ui_kit(array $install_state) {
   }
 }
 
+/**
+ * Method that calls another method to capture the installation start time.
+ */
+function acquia_cms_install_ui_kit_finished($install_state) {
```
Comment (vishalkhode1) [inline acquia_cms.profile:303]: @panshulK Why are we calling acquia_cms_install_ui_kit_finished() function two times:

1. `'finished' => 'acquia_cms_set_install_finished_time',`
2. `  $tasks['install_finished']['function'] = 'acquia_cms_install_ui_kit_finished';`

Comment (panshulK) [reply acquia_cms.profile:303]: @vishalkhode1 One is called for UI and another one is called when the CLI install runs.

File: src/Commands/SiteInstallCommands.php [hunk 3]
```diff
@@ -29,14 +30,19 @@ public function siteInstallPostCommand($result, CommandData $commandData) {
       $result = \Drupal::service('acquia_cms_common.utility')->rebuildSiteStudio();
       $this->yell('Finished rebuilding.');
       $this->setFinishedTime();
-      return is_array($result) && isset(array_shift($result)['error']) ? CommandResult::exitCode(self::EXIT_FAILURE) : CommandResult::exitCode(self::EXIT_SUCCESS);
     }
+    // Send data to telemetry based upon certain conditions.
+    if (\Drupal::service('module_handler')->moduleExists('acquia_telemetry') && Environment::isAhEnv()) {
+      acquia_cms_send_heartbeat_event();
```
Comment (vishalkhode1) [inline src/Commands/SiteInstallCommands.php:36]: @panshulK Please add a condition for function check. i.e
```suggestion
if (function_exists('acquia_cms_send_heartbeat_event')) {
  acquia_cms_send_heartbeat_event();
}
```

File: acquia_cms.profile (hunk 2 above)
Comment (vishalkhode1) [reply acquia_cms.profile:303]: Ah ok. Make sense.

File: src/Commands/SiteInstallCommands.php (hunk 3 above)
Comment (panshulK) [reply src/Commands/SiteInstallCommands.php]: Done, @vishalkhode1

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**